scripts/cot/botania/flowers/lexicon.zs

```diff
@@ -0,0 +1,286 @@
+import mods.botania.Lexicon;
```

Comment:
Please add explanation what this file do, and why its removing so many lexicon entries in form of comment.